lib/ansible/modules/files/copy.py

```diff
@@ -779,9 +779,8 @@ def main():
     if backup_file:
         res_args['backup_file'] = backup_file
 
-    module.params['dest'] = dest
```

Comment:
If there are any modules which accept both "path" and "dest" variables, but do *not* make them mutually exclusive in the parameter checking of the module, and it uses this method, *and* an end user is using different values for path and dest, this fix will result in changed behavior. (As previously path would be accidentally defaulted to when the load_file_common_arguments did a ".getParams(path, getParams(dest, None)).

I'd personally consider that an invalid end user use case that was relying on a bug if that is the case in any modules.

Reply:
The thing is that the behavior change only affects modules which have both which use this new feature, but none of the modules where I added this feature does that. So this PR itself cannot cause a behavior change. (Except for `ecs_certificate`, but there the implications should be no problem. The relevant thing that changes is which `path` is used in https://github.com/ansible/ansible/pull/61659/files#diff-90085fdcec6ed8b273ba885eaee60328R764, and for ecs_certificate, `path` is the certificate or the certificate's chain.)

Reply:
Ah, right, my mistake. I caught myself thinking that this change was to a copy utility method in /utility/ or something.